app/assets/javascripts/jquery.filmstrip-images.js

```diff
@@ -32,6 +32,12 @@
 
         $containerImgs.width(listingTotalWidth).height($filmstrip.data('thumb-height'));
 
+        // Change all data-alts to alts
+        $.each(imgs, function() {
+          $(this).prop('alt', $(this).data('alt'));
```

Comment:
Is it possible to cache the dom selector here? http://gregfranko.com/blog/jquery-best-practices/ "Caching DOM selectors"